data-raw/i18n_translations.R

```diff
@@ -91,3 +91,6 @@ translations_list <-
   map_depth(3, compact)
 
 saveRDS(translations_list, file = here("inst/i18n_translations"), version = 2)
```

Comment:
Could we also rename this file, too?

```suggestion
saveRDS(translations_list, file = here("inst/i18n_translations.rds"), version = 2)
```